src/index.js

```diff
@@ -0,0 +1,14 @@
+import './style.css';
+import scores from './scoresList.js';
+
+const tableBody = document.querySelector('tbody');
+function scoreElement(name, score) {
+  const tableRow = document.createElement('tr');
+  const tableData = document.createElement('td');
+
```

Comment:
## OPTIONAL 
* I strongly suggest using Es6 `arrow function` isntead of the `function` keyword as its a requirement to use Es6 syntax [Refer to the project requirement ](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md#:~:text=Learn%20how%20to%20use%20proper%20ES6%20syntax.)